05/app.js

```diff
@@ -9,20 +9,42 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const Listp = document.querySelectorAll("p");
+const Listlink = document.querySelectorAll("[href = '/dolor.html']");
+
+const countLinks = function(e) {
+    e.preventDefault();
+    stats.links["/dolor.html"]+=1;
+}
+    
+const countClicks = function(e) {
+    e.preventDefault();
+    if(e.target === e.currentTarget) {
+        stats.paragraphs.p1+=1;
```

Comment:
Zwróć uwagę, że tam są 3 paragrafy z różnymi id, trzeba wykorzystać coś takiego: `stats.paragraphs[id] += 1`, pamiętaj też aby najpierw sprawdzić czy taki paragraph istnieje już w `stats` - jeśli nie to go trzeba utworzyć.